### ISSUE TITLE
Has creating own Twitter app become neccessary again?

### ISSUE THREAD

**Issue by yutannihilation** [mkearney/rtweet#222]
Long ago I've asked about this question: https://github.com/mkearney/rtweet/issues/167

While I believe once I could use the app embedded in rtweet package, it seems not working now.

```r
library(httr)
oauth1.0_token(
  oauth_endpoints("twitter"),
  rtweet:::rtweet_app()
)
#> Error in init_oauth1.0(self$endpoint, self$app, permission = self$params$permission,  : 
#>   Unauthorized (HTTP 401).
```

The README still says

> NEW: All you need is a Twitter account and rtweet and you’re up and running!

But, is this not true anymore? Do we have to create our own app at last?

**Comment by mkearney** [comment]
Yeah. Twitter flagged my account and deactivated the embedded token. I think it would have been avoided if I proactively told them I was a developer, but I was just learning about everything and wasn't smart enough to do that. I've tried appealing their decision a couple of times but they don't seem to have any interest. I could use my own personal token (their decision didn't technically rule this out), but I don't want to risk being permanently black-listed. Unless I come up with a miracle solution, I'll have to revert defaults and documentation 😢

**Comment by mkearney** [comment]
The upside is that users must create their own token to have **write** and **DM** access and for any serious amounts of data collection it's advisable to have your own token anyway (more stable)...at least that's how I'm rationalizing it!

**Comment by yutannihilation** [comment]
Oh, thanks for the detail about that sad news...😢 
I saw many troubles even with famous web services that depend on Twitter on my country. So, your decision seems reasonable to me. I'm grad that I can still see you tweeting!

I wanted to confirm there are no hope with the embedded token because I've been wondered to withdraw my blog post that advatised about the token. So, thanks for your answer.

(I'll keep this issue open until the documents are reverted. But feel free to close this anytime!)
